### DISCUSSION TITLE
Is there better way to use "navigation.navigate" with typescript

### DISCUSSION THREAD

**Discussion by KoreanThinker** [react-navigation/react-navigation#10051] [Q&A]
Follow code working well. but this is so manual...

![image](https://user-images.githubusercontent.com/48207131/137267835-6ba55b36-d93e-4189-be0f-b5ff158dbd33.png)

**Comment by compojoom** [comment]
You could specify a default navigation prop as described here:
https://reactnavigation.org/docs/typescript/#specifying-default-types-for-usenavigation-link-ref-etc

This way you'll have less to type.
